src/python/py/models/builder.py

```diff
@@ -1735,6 +1834,25 @@ def make_gelu(self, layer_id, root_input, activation):
 
         return gelu_name
 
+    def make_relu(self, layer_id, root_input, activation):
+        relu_name = f"/model/layers.{layer_id}/mlp/act_fn/{activation}"
+        output = f"{relu_name}/output_0"
+        self.make_node(activation, inputs=[root_input], outputs=[output], name=relu_name, domain="")
+        self.make_value_info(output, self.io_dtype, shape=['batch_size', 'sequence_length', self.intermediate_size])
+        return relu_name
+
```

Comment:
As mentioned [here](https://github.com/microsoft/onnxruntime-genai/pull/958/files#r1796153035), Constant ops are auto-created. I think we can simplify this to the following.

```python
def make_pow(self, name, inputs, dtype, shape):
    output = f"{name}/output_0"
    self.make_node("Pow", inputs=inputs, outputs=[output], name=name)
    self.make_value_info(output, dtype, shape=shape)

def make_relu_squared(self, layer_id, root_input, activation):
    relu_name = self.make_relu(layer_id, root_input, activation="Relu")
    pow_name = f"/model/layers.{layer_id}/mlp/act_fn/Pow"
    pow_inputs = [f"{relu_name}/output_0", "/model/constants/TensorProto.INT64/1D/2"]
    pow_shape = ['batch_size', 'sequence_length', self.intermediate_size]
    self.make_pow(pow_name, pow_inputs, self.io_dtype, pow_shape)
    return pow_name
```